content/en/docs/security/config-best-practices.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Collector configuration best practices
+linkTitle: Collector config
+description: Best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When configuring the OpenTelemetry (OTel) Collector, consider the following
+practices to better secure your Collector instance.
+
+## Receivers and exporters
+
+We recommend enabling only the minimum required components for a Collector
+configuration: one receiver and one exporter. Configuring only the minimum set
+of components exposes a minimum set of required ports.
+
+- Receivers and exporters can be either push- or pull-based. In either case, the
+  connection established should be over a secure and authenticated channel.
+- Unused receivers and exporters should be disabled to minimize the attack
+  vector of the Collector.
+- Receivers and exporters might expose buffer, queue, payload, and worker
+  settings using configuration parameters. If these settings are available, you
+  should proceed with caution before modifying the default configuration values.
+  Improperly setting these values might expose the OpenTelemetry Collector to
+  additional attack vectors including resource exhaustion.
+
+### Safeguards against denial of service attacks
+
+Bind receivers' servers to addresses that limit connections to authorized users,
+so that your Collectors aren't exposed to the public internet or to wider
+networks than necessary.
+
+For example, if the OTLP receiver OTLP/gRPC server has only local clients, bind
+the `endpoint` setting to `localhost`:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
+```
+
+Use `localhost`-like addresses instead of `0.0.0.0`. For more information, see
+[CWE-1327: Binding to an Unrestricted IP Address](https://cwe.mitre.org/data/definitions/1327.html).
+
+To change the default endpoint to be `localhost`-bound in all components, enable
+the `component.UseLocalHostAsDefaultHost` feature gate. This feature gate will
+be enabled by default in the Collector in a future release.
+
+### Encryption and authentication
+
+Your OTel Collector configuration should include encryption and authentication.
+
+- For communication encryption, see
+  [Configuring certificates](/collector/configuration/#setting-up-certificates).
+- For authentication, use the OTel Collector's authentication mechanism, as
+  described in [Authentication](/collector/configuration/#authentication).
+
+## Processors
+
+[Processors](/collector/configuration/#processors) sit between receivers and
+exporters. They are responsible for processing telemetry before it's analyzed.
+From a security perspective, processors are useful in a few ways.
+
+### Recommended processors
```

Comment:
I don't think we should prescribe which processors people should use. We could provide some example configurations for common use-cases though.

Reply:
Removed the section in favor of examples.